AppCenterAuth/AppCenterAuth/Internals/Models/MSAuthConfig.m

```diff
@@ -47,10 +45,10 @@ - (BOOL)isValid {
 - (BOOL)areAuthoritiesValid {
   BOOL foundDefault = NO;
   for (MSAuthority *authority in self.authorities) {
-    if (![authority isValid]) {
```

Comment:
Is the `!` removed intentionally?

Reply:
No it was intentional I will fix it.

Reply:
No it is not. But thanks for catching this.